### DISCUSSION TITLE
batch processing of material is required

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#9607] [Editor]
### Describe the project you are working on

ordinary RPG game

### Describe the problem or limitation you are having in your project

if i select multiple materials and change the values, only one material value changes.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

multiple material values and be changed simultaneously.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

.

### Is there a reason why this should be core and not an add-on in the asset library?

if there is no improvement, there is the inconvenience of having 
to change various material values dozens of times  repeatedly

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.
